### PR TITLE
Removes Pipelineruns from tests folder

### PR DESCRIPTION
**Pull request by ashishranjan738**
Description of changes:

This commit removes pipelines runs for test folder.

### CODE REVIEW

File: tests/README.md [hunk 1]
```diff
@@ -67,7 +67,106 @@ EOF
 ```
 
 ### Scheduling recurrent test runs.
-For scheduling recurrent test runs a cron job is used to trigger the pipeline from the pipelinerun template. Reffer to the for a sample cron job and pipelinerun temaple. [example.yaml](pipelineruns%2Feks%2Fexample.yaml)
+For scheduling recurrent test runs a cron job is used to trigger the pipeline from the pipelinerun template. Refer to the for a sample cron job and pipelinerun template below.
```
Comment (hakuna-matatah) [inline tests/README.md:70]: nit: `Refer "to the for" a sample cron job and pipelinerun template below.` ->  `Please refer to below example for sample cron job and pipelinerun template`

Comment (hakuna-matatah) [reply tests/README.md:70]: nit: `For scheduling recurrent test runs a cron job is used to trigger the pipeline from the pipelinerun template` --> `To schedule recurrent test runs, you can leverage cron-job to trigger a pipeline run using trigger-template`

Comment (ashishranjan738) [reply tests/README.md]: done

**Review by hakuna-matatah — APPROVED** [review]
lgtm

**Pull request merged**